packages/now-python/test/fixtures/14-unicode-handler/probe.js

```diff
@@ -0,0 +1,34 @@
+const fetch = require('../../../../../test/lib/deployment/fetch-retry');
+
+async function fetchDeploymentUrl(url, opts) {
+  for (let i = 0; i < 50; i += 1) {
+    const resp = await fetch(url, opts);
+    const text = await resp.text();
+    if (text && !text.includes('Join Free')) {
+      return { resp, text };
+    }
+
+    await new Promise(r => setTimeout(r, 1000));
+  }
+
+  throw new Error(`Failed to wait for deployment READY. Url is ${url}`);
+}
+
+module.exports = async function({ deploymentUrl, fetch, randomness }) {
+  const nowjson = require('./now.json');
+  const probe = nowjson.probes[0];
+  const probeUrl = `https://${deploymentUrl}${probe.path}`;
+  const { resp, text } = await fetchDeploymentUrl(probeUrl, {
```

Comment:
Can you use `fetch(deploymentUrl)` here instead?

```js
const res = await fetch(deploymentUrl)
const respBody = await res.json();
```

Then you don't need this additional function, right?

Reply:
I could, but the other tests that use **mustContain** use that function so I was trying to stay consistent.

Reply:
Take a look at the few tests that currently use `probe.js` instead of the `probes` key in now.json

https://github.com/zeit/now/blob/32aa94bf3d186bbe88546643b5bb5688b97f8d0e/packages/now-node/test/fixtures/07-content-length/probe.js#L3

I think you can do something similar here.

Reply:
Done. Thanks.